includes/collection/class-replies.php

```diff
@@ -171,10 +173,16 @@ public static function get_context_collection( $post_id ) {
 		$post_uri = ( new PostTransformer( $post ) )->to_id();
 		\array_unshift( $ids, $post_uri );
 
+		if ( is_user_disabled( $post->post_author ) ) {
+			$author = new Blog();
+		} else {
+			$author = Actors::get_by_id( $post->post_author );
+		}
```

Comment:
```suggestion
		$author = Actors::get_by_id( $post->post_author );
		if ( is_wp_error( $author ) ) {
			$author = new Blog();
		}
```
Would this be better?